src/clr-angular/utils/drag-and-drop/providers/drag-handle-registrar.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Injectable, Renderer2} from "@angular/core";
+
+import {ClrDragEventListener} from "./drag-event-listener";
+
+// This provider registers the drag handle element.
+// When it registers a element as a drag handle, it attaches that element to the listeners from ClrDragEventListener.
+// Also, it adds the "drag-handle" css class to the registered element through Renderer.
+@Injectable()
+export class ClrDragHandleRegistrar<T> {
+    private _customHandleEl: Node;
+    private _defaultHandleEl: Node;
+
+    get defaultHandleEl() {
+        return this._defaultHandleEl;
+    }
+
+    set defaultHandleEl(value: Node) {
```

Comment:
You are using three different naming conventions for the parameters: 
`set defaultHandleEl(value: Node)`
`private makeElementHandle(el: Node)`
`public registerCustomHandle(handleElement: Node)`

Lets just choose one? :D

Reply:
Will address this issue.